src/gui/editors/PianoRoll.cpp

```diff
@@ -2863,6 +2870,22 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 
 	bool show32nds = ( m_zoomingModel.value() > 3 );
 
+	// alternating shades for better contrast
+	// count the bars which disappear on left by scrolling
+	int barCount = m_currentPosition / MidiTime::ticksPerTact();
+	int leftBars = m_currentPosition / m_ppt;
+
+	p.setBrush( backgroundShade() );
+
+	for ( int x = WHITE_KEY_WIDTH; x < width() + m_currentPosition; x += m_ppt, ++barCount )
+	{
+		if ( (barCount + leftBars)  % 2 != 0 )
+		{
+				p.drawRect( x - m_currentPosition, PR_TOP_MARGIN, m_ppt,
```

Comment:
Okay, I've got a green line on top of the shading rect:
![2016-10-06-143342_1920x1080_scrot](https://cloud.githubusercontent.com/assets/6502580/19152379/00306bf0-8bd2-11e6-852c-cfeaa2835b45.png)

What helps is:
```
p.fillRect( x - m_currentPosition, PR_TOP_MARGIN, m_ppt,
					height() - ( PR_BOTTOM_MARGIN + PR_TOP_MARGIN ), backgroundShade() );
```

by removing the : ```p.setBrush( backgroundShade() );``` of course. 


```